tenable/io/v3/mssp/logos/api.py

```diff
@@ -0,0 +1,229 @@
+'''
+Logos
+=====
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`Logos <io-mssp-logos>` API endpoints.
+
+Methods available on ``tio.v3.mssp.logos``:
+
+.. rst-class:: hide-signature
+.. autoclass:: LogosAPI
+    :members:
+'''
+from io import BytesIO
+from typing import BinaryIO, Dict, List, Optional, Union
+from uuid import UUID
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+from tenable.io.v3.mssp.logos.schema import LogoSchema
+
+
+class LogosAPI(ExploreBaseEndpoint):
+    '''
+    This class contains methods related to Logos API
+    '''
+    _path = 'api/v3/mssp/logos'
+    _conv_json = True
+    _schema = LogoSchema()
+
+    def add(self, logo: Union[BytesIO, BinaryIO], name: str) -> UUID:
+        '''
+        Adds a new logo
+
+        Args:
+            fobj (FileObject, optional):
+                A file-like object of the logo.
+            name (str):
+                Name of the logo.
+
+        Returns:
+            :obj:`str`:
+                Return the logo_id.
+
+        Example:
+            >>>> with open('/opt/download1.png', 'rb') as fobj:
+            ....    resp = tio.v3.mssp.logos.add(fobj, 'logo1')
+        '''
+        payload = {'files': {'logo': logo, 'name': name}}
+        return self._post(**payload)['id']
+
+    def list(self):
+        '''
+        Returns list of logos
+
+        Args:
+
+        Returns:
+            :obj:`list`:
+                Returns all the logo details as a list.
+
+        Example:
+            >>>> tio.v3.mssp.logos.list()
+        '''
+        return self._get()['logos']
+
+    def search(self):
+        '''
+        Get the listing of Logos from MSSP portal.
```

Comment:
Please use this doc string as a reference for search endpoints
https://github.com/dhvanilshah-tenable/pyTenable/pull/12/files#diff-f08a9da0fb05c1c53ae0da456a95945971dfbf2362c0fa30e541363d3728e227R47